### DISCUSSION TITLE
cross-site cookies

### DISCUSSION THREAD

**Discussion by GimmeDataNow** [FreeTubeApp/FreeTube#5313] [Q&A]
I just wanted to ask if the [https://goo.gle/3pcd-dev-issue](google cookie issue) is of any concern to the functionality of Freetube?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
It could maybe affect the API's that we are using. We haven't experience issues with it so far. Some investigating is still needed as only 1% of the stable clients is affected by this it and we will be on the lookout when more of the changes are starting to roll out.
